pallets/attestation/src/lib.rs

```diff
@@ -230,6 +239,7 @@ pub mod pallet {
 			// the original attester
 			if attestation.attester != revoker {
 				let delegation_id = attestation.delegation_id.ok_or(Error::<T>::UnauthorizedRevocation)?;
+				let max_parent_checks = max_parent_checks.min(T::MaxParentChecks::get());
```

Comment:
maybe we should ensure the maximum in the beginning and fail?

Reply:
So we just remove the user-provided parameter altogether? It would make sense in that case.

Reply:
Yes we could do that. 🤔 It's probably the correct thing to do.

It's just cheaper for the user, because he doesn't have to pay the worst case. I think the extrinsic is quite expensive so maybe that makes a difference.

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/189/commits/0ddb538a40f37949049b516c3fa96c4f938e533b